docs/user_guide/using_the_library.rst

```diff
@@ -32,6 +32,16 @@ The most basic function to use when inspecting a single NWBFile is the
 
 This returns a ``list`` of :py:class:`~nwbinspector.register_checks.InspectorMessage` objects.
 
+If you have an ``NWBFile`` object in memory, you can run:
+
+.. code-block:: python
+
+    from nwbinspector import available_checks, run_checks
```

Comment:
Also I just checked - if you want to use the import like this, will have to update the outer exposure to include `run_checks` alongside the other `inspect_*` functions